CMakeLists.txt

```diff
@@ -110,7 +110,10 @@ target_link_libraries(${PROJECT_NAME}
   ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
   ${QT_LIBRARIES} ${catkin_LIBRARIES} ${Boost_LIBRARIES} log4cxx)
 
-install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
+add_executable(${PROJECT_NAME}_updater src/collisions_updater.cpp )
```

Comment:
how about moveit_collision_updater - I'm fairly sure there is no conflict there (grep confirms)